src/modules/configs/templates/templates.py

```diff
@@ -0,0 +1,229 @@
+final = {
```

Comment:
Templates should be divided into: final, merged, application, project, operator. Vars should be named something generic e.g. `config` so in the module we can do something like
```
temp = templates.final
final = temp.config
```